tools/image_processing/imagej2/imagej2_watershed_binary/imagej2_watershed_binary.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_watershed_binary" name="Watershed segmentation" version="1.0.0">
+    <description>of binary image with ImageJ2</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="fiji_20141125_requirements" />
```

Comment:
General comment: I think this should be renamed to something without a version number. This would have the advantage that we don't need to change every tool if we increase the dependency.
Similar to the tool-version number. This should also move into a macro similar to: https://github.com/galaxyproject/tools-iuc/blob/master/tools/bedtools/macros.xml#L8

This will make migration to new fiji versions easy :)